akka-cluster-metrics/src/main/protobuf/ClusterMetricsMessages.proto

```diff
@@ -2,6 +2,8 @@
  * Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
  */
 
+syntax = "proto2";
```

Comment:
protoc3 can still parse protoc2 files but the code generation will be new, not the old version